Algorithms/Divide-and-Conquer/Weighted Job Scheduling.cpp

```diff
@@ -0,0 +1,59 @@
+#include <iostream>
+#include <algorithm>
+using namespace std;
+
+struct Job{
+    int start, finish, profit;
+};
+
+bool compareFinishTime(Job s1, Job s2){
+    return (s2.finish > s1.finish);
+}
+
+int binarySearch(Job jobs[], int index){// to find the latest job (before current job) that doesn't conflict with current job.
```

Comment:
Please follow to [Coding standards](https://github.com/sahilbansal17/Competitive_Coding/wiki/Coding-Standards) for comments. Comments should be above the code statements.